web-animations-2/Overview.bs

```diff
@@ -357,29 +395,37 @@ an animation, <var>animation</var>, to <var>seek time</var> is as follows:
 
     1.   Abort these steps.
 
-1.  Update either <var>animation</var>'s <a>hold time</a> or
-    [=start time=] as follows:
+    1.  Let <var>valid seek time</var> be the result of running the
+        [=validate a CSSNumberish time=] procedure with <var>seek time</var>
+        as the input.
 
-    <dl class="switch">
+    1.  If <var>valid seek time</var> is false, abort this procedure.
 
-        :   If <em>any</em> of the following conditions are true:
+    1.  Update either <var>animation</var>'s <a>hold time</a> or
+        [=start time=] as follows:
 
-            *   <var>animation</var>'s <a>hold time</a> is
-                <a lt="unresolved">resolved</a>, or
-            *   <var>animation</var>'s [=start time=]
-                is <a lt="unresolved">unresolved</a>, or
-            *   <var>animation</var> has no associated <a>timeline</a> or the
-                associated <a>timeline</a> is
-                <a lt="inactive timeline">inactive</a>, or
-            *   <var>animation</var>'s [=playback rate=] is 0,
+        <dl class="switch">
 
-        ::  Set <var>animation</var>'s <a>hold time</a> to <var>seek time</var>.
+            :   If <em>any</em> of the following conditions are true:
 
-        :   Otherwise,
-        ::  Set <var>animation</var>'s [=start time=] to the result of evaluating
-            <code>|timeline time| - (|seek time| / [=playback rate=])</code>
-            where <var>timeline time</var> is the current <a>time value</a>
-            of <a>timeline</a> associated with <var>animation</var>.
+                *   <var>animation</var>'s <a>hold time</a> is
+                    <a lt="unresolved">resolved</a>, or
+                *   <var>animation</var>'s [=start time=]
+                    is <a lt="unresolved">unresolved</a>, or
+                *   <var>animation</var> has no associated <a>timeline</a> or
+                    the associated <a>timeline</a> is
+                    <a lt="inactive timeline">inactive</a>, or
+                *   <var>animation</var>'s [=playback rate=] is 0,
+
+            ::  Set <var>animation</var>'s <a>hold time</a> to
+                <var>seek time</var>.
+
+            :   Otherwise,
+            ::  Set <var>animation</var>'s [=start time=] to the result of
+                evaluating
+                <code>|timeline time| - (|seek time| / [=playback rate=])</code>
+                where <var>timeline time</var> is the current <a>time value</a>
+                of <a>timeline</a> associated with <var>animation</var>.
 
     </dl>
```

Comment:
Looks like the indentation level here doesn't match the opening tag.

Reply:
Fixed.